### DISCUSSION TITLE
GitHub Student Developer Pack :: does not recognize (We need a little more proof to verify your academic status.)

### DISCUSSION THREAD

**Discussion by dabzse** [community/community#127069] [GitHub Education]
### Hi everyone,

png/jpg/jpeg, 1024x768 (or more pixel), between approximately 600-950 kB in size.
once I was able to made it, in February 2022, but now I'm trying it to re-new since March 2024.
whom I need to write and send/upload the photo/scan of my youth card? because this system can't recognize this youth card is not an american one...

thank you!

**Comment by math-silva** [comment]
**Hi there,**

Here are a few things that might help:

Make sure your information is up-to-date: Double-check that your GitHub profile includes your current school email address. This is often a key factor in the verification process. The photo of your youth card should be recent and clear. If it's older, try taking a new one. Usually, the verification process is pretty quick, often taking just a few hours. If it's been longer, you might want to reach out to GitHub Support.

Don't worry about your card not being American. I'm from Brazil, and I was able to verify my student status using a photo of my enrollment proof from my Brazilian university. 

**I hope this helps! If it does, please mark this response as accepted.**

**Reply by dabzse** [reply]
if in 2022 was okay, and now since March I made few times double-check, the university's e-mail address is working and again confirmed....

gh wants to see the expiration date, it will expire in 2028, but the university's name has been changed a little bit. but during my youth card is valid, I'm not eligible for new one.
the photos and the scans from both sides are new / almost new. and the stamp is visible properly that I am an active student....

about the enrollment proof: the system had (or still has) some bugs, so not everything is visible properly. some information is on the top of each other (the system wasn't prepare for longer enumeration) :/

I tried to reach the gh support, but they no longer accepts these type of questions; their answer: there is a community.

but anyway, thank you! will try again with the enrollment proof, hopefully will accept it
